Sources/SnowManLogic/ViewLogic/View.swift

```diff
@@ -0,0 +1,52 @@
+public class View: Viewable {
+	var io: Interactable
+
+	public init(io: Interactable) {
+		self.io = io
+	}
+
+	public func receiveGuess() -> String {
+		var guess = io.getUserInput()
+		while (guess == "" || !(containsOnlyLetters(guess))) {
+			io.display("Invalid guess. Please enter a lowercase letter or word.")
+			guess = io.getUserInput()
+		} 
+		return guess
+	}
+
+	public func assignBlanks(gameWord: String, correctGuesses: [String]) {
+		var blanks: [String] = []
+		let letters = gameWord.characters.map { String($0) }
+		
+		for letter in letters { 
+			if (correctGuesses.contains(letter)) { 
+	           		blanks.append(letter)  
+       			} else { 
+           			blanks.append("__ ") 
+       			} 
+   		}
+   		let assignedBlanks = blanks.joinWithSeparator(" ")
+   		io.display(assignedBlanks)
+	}
```

Comment:
I removed return statements from each of these methods except the first. Please check to make sure I am testing this correctly in the `ViewSpec`